src/views/Predictions/helpers.ts

```diff
@@ -1,15 +1,44 @@
-import { ethers } from 'ethers'
+import { BigNumber, ethers } from 'ethers'
 import { NodeRound } from 'state/types'
 import { formatBigNumberToFixed } from 'utils/formatBalance'
 import getTimePeriods from 'utils/getTimePeriods'
 
-export const formatUsdv2 = (usd: ethers.BigNumber) => {
-  return `$${formatBigNumberToFixed(usd, 3, 8)}`
+const MIN_PRICE_USD_DISPLAYED = BigNumber.from(100000)
+const MIN_PRICE_BNB_DISPLAYED = BigNumber.from('1000000000000000')
+const DISPLAYED_DECIMALS = 3
+
+type formatPriceDifferenceProps = {
+  price?: BigNumber
+  minPriceDisplayed: BigNumber
+  unitPrefix: string
+  decimals: number
+}
+
+const formatPriceDifference = ({
+  price = BigNumber.from(0),
+  minPriceDisplayed,
+  unitPrefix,
+  decimals,
+}: formatPriceDifferenceProps) => {
+  const value = price
```

Comment:
Any reason to instantiate this variable and not use `price` directly?

Reply:
Good point, it was used for doing a default assignment before. Since it's done in the params now I will clean this.